tests/test_sqla_yaml_fixtures.py

```diff
@@ -499,3 +526,27 @@ def test_custom_loader(session):
     assert users[0].email == 'deedee@ramones.org'
     assert users[1].username == 'joey'
     assert users[1].email == 'joey@ramones.org'
+
+
+def test_many_2_many(session):
+    fixture = """
+- User:
+  - __key__: joey
+    username: joey
+    email: joey@example.com
+  - __key__: bobby
+    username: bobby
+    email: bobby@example.com
+  - __key__: johnny
+    username: johnny
+    email: johnny@example.com
+    friends:
+      - joey
+      - bobby
+"""
+    sqla_yaml_fixtures.load(BaseModel, session, fixture)
+    users = session.query(User).order_by(User.username).all()
+    assert users[0].friends == []
+    assert users[1].friends == []
+    assert users[2].friends[0].id == users[0].id
```

Comment:
just a nitpick. I read this and I need to think which friend will be the last one.
One way to improve it would be just add one assert on the `users[2].username`, just to make it clearer.

Reply:
updated commit